piccolo/columns/column_types.py

```diff
@@ -1217,11 +1224,27 @@ def set_proxy_columns(self):
         the ``ForeignKey`` column for each column in the table being pointed
         to.
         """
-        _foreign_key_meta = object.__getattribute__(self, "_foreign_key_meta")
-        for column in _foreign_key_meta.resolved_references._meta.columns:
+        _fk_meta = object.__getattribute__(self, "_foreign_key_meta")
+        for column in _fk_meta.resolved_references._meta.columns:
             _column: Column = column.copy()
             setattr(self, _column._meta.name, _column)
-            _foreign_key_meta.proxy_columns.append(_column)
+            _fk_meta.proxy_columns.append(_column)
+
+        def all_columns():
+            """
+            Allow a user to access all of the columns on the related table.
+
+            For example:
+
+            Band.select(Band.name, *Band.manager.all_columns()).run_sync()
+
+            """
+            return [
+                getattr(self, column._meta.name)
+                for column in _fk_meta.resolved_references._meta.columns
+            ]
+
+        setattr(self, "all_columns", all_columns)
```

Comment:
@dantownsend I think this looks great and reduces verbosity in queries.

Reply:
@sinisaos Awesome, thanks!